### ISSUE TITLE
windows/sound.cpp: Rename CheckDeviceCapabilities()

### ISSUE THREAD

**Issue by hoffie** [jamulussoftware/jamulus#1409]
**Describe the bug**
`CheckDeviceCapabilities()` in `windows/sound.cpp` does more than what the name suggests: It has side effects such as updating instance variables such as `lNumInChan`.

**Expected behavior**

Have a name which makes it clear that updates happen.

**Version of Jamulus**
d5d4691221b76bc648d605dac867aeaa561d0a52

cc @ann0see @pljones

**Comment by DominikSchaller** [comment]
Maybe a stupid question, but what does "good first issue" mean? Does it mean, it's a good issue for a new developer?

**Comment by hoffie** [comment]
> what does "good first issue" mean? Does it mean, it's a good issue for a new developer?

Yes, this was the intention. I think this issue requires little Jamulus-specific context but could be worked on by some developer who is new to the project.

**Comment by ann0see** [comment]
Any news here? @DominikSchaller @henkdegroot do you want to rename the function?

**Comment by pgScorpio** [comment]
@ann0see 

> Any news here? @DominikSchaller @henkdegroot or @pgScorpio (sorry for the pings...) do you want to rename the function?

No, just wait for the sound redesign.... It's already changed there.

**Comment by ann0see** [comment]
Thanks!

I believe it's still relevant. 
Feel free to raise a PR - but be aware that the review process might take some time. 

Concerning contributing: make sure to read the CONTRIBUTING.md file.

**Comment by T-Roel** [comment]
@ann0see and @softins thank you very much for your feedback! Since it doesn't seem necessary to do this issue, could you recommend another good first one? 

@softins I'm starting using Jamulus in order to investigate some cognitive aspects in networked music performance as part of my PhD. I don't use Discord, but I intend to create a profile to chat with the community :)

**Comment by softins** [comment]
@T-Roel cool! I've been told the Discord is currently only open to established team members - my bad! But I've just found you on Facebook, and I will make contact with you on Messenger to chat a bit.
